classes/class-kp-logger.php

```diff
@@ -29,10 +29,7 @@ public static function log( $data ) {
 		$kp_settings = get_option( 'woocommerce_klarna_payments_settings', array() );
 		if ( 'no' !== $kp_settings['logging'] ) {
 			$message = self::format_data( $data );
-			if ( empty( self::$log ) ) {
-				self::$log = new WC_Logger();
-			}
-			self::$log->add( 'klarna_payments', wp_json_encode( $message ) );
+			KP_WC()->logger()->debug( wp_json_encode( $message ) );
 		}
 
 		if ( isset( $data['response']['code'] ) && ( $data['response']['code'] < 200 || $data['response']['code'] > 299 ) ) {
```

Comment:
Question: I see we still have the logic here to log errors to the option for the system report, but isn't that replaced by the package as well? Do we still need this code here?